docs/reference/buildx_prune.md

```diff
@@ -21,6 +21,18 @@ Remove build cache
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Clears the build cache of the selected builder.
+
+You can finely control what cache data is kept using:
+
+- The `--filter=until=<duration>` flag to keep images that have been used in
+  the last `<duration>` time.
+- The `--keep-storage=<space>` flag to keep `<space>` bytes of data in the cache.
+- The `--all` flag to allow clearing internal helper images and frontend images
+  set using the `#syntax=` directive or the `BUILDKIT_SYNTAX` build argument.
+
```

Comment:
I think we should also carry some bits from https://github.com/docker/cli/pull/2864

Reply:
Let's do this in a follow-up, maybe adding that info into a new guide around cache management?

Would be a good place to tie together `prune` and `du`, though not sure how we'd link in the GC policy, I think that's managed by the docker daemon?